### DISCUSSION TITLE
【摄影】青衣前辈真是小巧玲珑！

### DISCUSSION THREAD

**Discussion by Jack1Nero1** [share121/inter-knot#2231] [其他]
![20240815152110x](https://github.com/user-attachments/assets/97d092ee-c36c-4256-873d-863f5ca2570a)
今天去火锅店参加吃辣挑战的时候发现青衣前辈也在，但是旁边站着个穿青灰色外套的灰发难，那到底是谁啊？青衣前辈不会有对象了吧？那种事情不要啊！

**Comment by Jack1Nero1** [comment]
*纠正一下错别字，灰发男，还有谁知道那个人到底是谁啊呃呃啊

**Comment by Zzz-HK** [comment]
好像是六分街的麦片兄妹里的哥哥，上次路过六分街的时候瞅了一眼，妹妹挺可爱的，就是哥哥太路人了（笑）

**Comment by Kamisato100** [comment]
![20240814183039](https://github.com/user-attachments/assets/1014763a-b864-4fa2-b4ca-5e78a652a530)
小小只的也很可爱<img src="https://bbs-static.miyoushe.com/static/2023/11/16/a6976b54880a512092a19f34c1cd3931_2328184749682335286.png" alt="铃-萌混过关" width="75" >
